### PR TITLE
Adds ability to get current video time

### LINKED ISSUE #259: Add a function that returns current video time in Video.js
Currently `Synchronizer.js` is using window to access the video player but with the addition of the `Video` class this is no longer possible

### PR DESCRIPTION
**Pull request by jaysaleh**
This pr fixes #259. Adds a function to the `Video` class that returns the current video time.

### CODE REVIEW

File: src/main/webapp/view/video/video.js [hunk 1]
```diff
@@ -49,6 +47,11 @@ export default class Video {
     startVideoSyncTimer();
   }
 
+  /** Returns current video time of 'videoPlayer'. */
+  getCurrentVideoTime() {
```
Comment (n-wach) [inline src/main/webapp/view/video/video.js:51]: Can you include the units?

Comment (n-wach) [reply src/main/webapp/view/video/video.js:51]: getCurrentVideoTimeSeconds

File: src/main/webapp/view/video/video.js [hunk 2]
```diff
@@ -50,6 +48,11 @@ export default class Video {
     startVideoSyncTimer();
   }
 
+  /** Returns current video time of 'videoPlayer'. */
+  getCurrentVideoTimeSeconds() {
```
Comment (billie-cat) [inline src/main/webapp/view/video/video.js:52]: Seems error prone to have some times be in ms and some be sec. Shouldn't everything be in ms?

Comment (jaysaleh) [reply src/main/webapp/view/video/video.js:53]: This is being fixed by @n-wach

**Pull request merged**